File: app/src/main/java/com/example/pasteleria/model/Usuario.java
package com.example.pasteleria.model;

data class Usuario {
    val nombre: String,
    val correo: String,
    val password: String,
    val telefono: String?,
    val direccion: String,
    val edad: Int,
    val descuento: Int,
    val beneficios: List<String>,
    val fechaRegistro: String
)

File: app/src/main/java/com/example/pasteleria/model/Usuario.kt
package com.example.pasteleria.model

data class Usuario(
    val nombre: String,
    val correo: String,
    val password: String,
    val telefono: String? = null,
    val direccion: String,
    val edad: Int,
    val descuento: Int = 0,
    val beneficios: List<String> = emptyList(),
    val fechaRegistro: String
)